datasets/pcfg/data_generation.py

```diff
@@ -149,26 +148,31 @@ def main(argv: Sequence[str]):
     
     out_file_src = open(data_path + 'it_dec_val.src', 'w')
     out_file_tgt = open(data_path + 'it_dec_val.tgt', 'w')
```

Comment:
Sorry for not catching this earlier. I Was struggling to understand the code below, since I did not understand what "out_file_src_dec" stood for. If I understand correctly "out_file_src" is for the "validation set" and "out_file_src_dec" for the test set, right? Maybe we should call them "out_file_src_validation" and "out_file_src_test" instead to make it more readable? (same with *_tgt)